settings/volare-file-settings.json

```diff
@@ -5,15 +5,15 @@
     "type": "22",
     "uuid": "urn:mrn:signalk:uuid:c0d79334-4e25-4245-8892-54e8ccc8021d"
   },
-
   "pipedProviders": [
     {
       "id": "nmeaFromFile",
       "pipeElements": [
         {
           "type": "providers/filestream",
           "options": {
-            "filename": "samples/plaka.log"
+            "filename": "samples/plaka.log",
+            "providerId": "nmeaFromFile"
```

Comment:
Why the unrelated changes?

Reply:
Not sure, actually. I didn't touch those files. Could have been a result of d2fe5e2

Reply:
I don't think so. If you had this in the shared org repo I could fix small stuff like this myself instead of writing about it...

Reply:
Not really sure what the issue is here, btw. File seems unchanged to me